frontend/app/brains-management/[brainId]/components/BrainManagementTabs/components/SettingsTab/SettingsTab.tsx

```diff
@@ -37,11 +38,26 @@ export const SettingsTab = ({ brainId }: SettingsTabProps): JSX.Element => {
     setValue,
     dirtyFields,
     resetField,
-    pickPublicPrompt,
+    setIsUpdating,
     promptId,
-    removeBrainPrompt,
+    getValues,
+    reset,
+    updateFormValues,
   } = useSettingsTab({ brainId });
 
+  const promptProps = {
+    brainId,
+    getValues,
+    promptId,
+    register,
+    reset,
+    setValue,
+    resetField,
+    updateFormValues,
+    dirtyFields,
+    setIsUpdating,
+  };
+
```

Comment:
Since many values seem to concern form utils, maybe we can use `useFormContext` instead? (low logaf)